certified-connectors/JIRA/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2183 @@
+{
+    "swagger": "2.0",
+    "info":
+    {
+        "x-ms-deployment-version": "1.0.0",
+        "version": "1.0.0",
+        "title": "JIRA",
+        "description": "JIRA is a software development tool for agile teams to plan, track, and release world-class software. Connecting JIRA issues to the rest of your tools helps break down barriers and unleash the potential of your team.",
+        "x-ms-api-annotation":
+        {
+            "status": "Production"
+        },
+        "contact": 
+        {
+            "name": "Microsoft",
+            "url": "https://support.microsoft.com"
+        }
+    },
+    "host": "notused.tobereplaced.net",
+    "basePath": "/rest/api",
+    "schemes":
+    [
+        "https"
+    ],
+    "produces":
+    [
+        "application/json"
+    ],
+    "paths":
+    {
+        "/3/issue/{issueIdOrKey}":
+        {
+            "put":
```

Comment:
@depfeife-msft Indentation - All the properties are not aligned properly, please refer [this](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/365Training/apiDefinition.swagger.json) swagger for alignment.